src/lib/process/_tests_/fixtures.ts

```diff
@@ -1,45 +1,61 @@
-export const restrictionsBeforeMap = JSON.stringify({
-  SenderOwnsAllInputs: [],
-  SenderHasInputRole: [
-    {
-      index: 0,
-      roleKey: 'Supplier',
-    },
-  ],
-  FixedOutputMetadataValueType: [
-    {
-      index: 0,
-      metadataKey: 'SomeMetadataKey',
-      metadataValueType: 'Literal',
-    },
-    {
-      index: 0,
-      metadataKey: 'SomeOtherMetadataKey',
-      metadataValueType: 'File',
-    },
-  ],
-})
+export const restrictionsBeforeMap = JSON.stringify([
+  { SenderOwnsAllInputs: [] },
+  {
+    SenderHasInputRole: [
+      {
+        index: 0,
+        roleKey: 'Supplier',
+      },
+    ]
+  },
+  { op: 'and' },
+  {
+    FixedOutputMetadataValueType: [
+      {
+        index: 0,
+        metadataKey: 'SomeMetadataKey',
+        metadataValueType: 'Literal',
+      },
+      {
+        index: 0,
+        metadataKey: 'SomeOtherMetadataKey',
+        metadataValueType: 'File',
+      },
+    ]
+  },
+  { op: 'and' },
+  { op: 'and' },
+])
 
 export const restrictionsAfterMap = [
```

Comment:
Is this still used?

Reply:
not sure, will double check, thanks for flagging.

Reply:
Removed.